pkg_resources/_vendor/pyparsing.py

```diff
@@ -766,11 +794,14 @@ def nullDebugAction(*args):
 
 # this version is Python 2.x-3.x cross-compatible
 'decorator to trim function calls to match the arity of the target'
+

```

Comment:
I'm pretty sure this string should be the do string for the function. Perhaps also the comment should be moved inside.